services/user-service/actions/auth-action/facebook/redirect-action.ts

```diff
@@ -0,0 +1,16 @@
+import express, {Request, Response} from "express";
+import passport from "passport";
+import { signAccessToken } from "../../../utils/token";
+
+const router = express.Router();
+
+router.get("/",  
+  passport.authenticate('facebook', { failureRedirect: '/', failureMessage: true }), 
+  (req, res) => {
+    const token = signAccessToken({id: (req.user as any).id});
+    let redirectUrl = process.env.USER_SERVICE_FACEBOOK_REDIRECTION_URL as string;
+    redirectUrl = `${redirectUrl}/${(req.user as any).id}/${token}`; 
```

Comment:
use the correct user type definition, we need to avoid the usage of `any` as much as possible